server/statistics/kind_test.go

```diff
@@ -27,12 +27,16 @@ type testRegionInfoSuite struct{}
 
 func (s *testRegionInfoSuite) TestGetLoads(c *C) {
 	queryStats := &pdpb.QueryStats{
-		Get:         5,
-		Coprocessor: 6,
-		Scan:        7,
-		Put:         8,
-		Delete:      9,
-		DeleteRange: 10,
+		Get:                    5,
```

Comment:
the test is not same with the state set??

Reply:
only set different num